Misc/NEWS.d/next/Library/2019-11-13-07-37-11.bpo-38731.9qmcSx.rst

```diff
@@ -0,0 +1 @@
+Fix a crash during error handling in :meth:`py_compile.main`.
```

Comment:
IMO `crash` is a strong word that implies a CPython segfault.
`unhandled exception` is more adequate :slightly_smiling_face: